test/test_type_hints.py

```diff
@@ -63,12 +63,6 @@ def get_all_examples():
     """
     blacklist = {
         "_np",
-        "refine_names",
-        "rename",
-        "names",
-        "align_as",
-        "align_to",
-        "unflatten",
```

Comment:
To summarize:
- names just works
- align_as just works
- This PR adds manual entries for unflatten, refine_names, and rename 
- `align_to` works from changing the annotations generation code

Is that an accurate summary of what is going on in this PR?

Reply:
Your summary is right on.

I'll wait for feedback on what you'd like to do with the annotation generation code with regard to `align_to`.  I'm happy to strip-out the changes to the generator and add the manual case for `align_to` if you think that approach is preferable.